kornia/augmentation/random_generator/_2d/cutmix.py

```diff
@@ -59,27 +60,25 @@ def __repr__(self) -> str:
         repr = f"cut_size={self.cut_size}, beta={self.beta}, num_mix={self.num_mix}"
         return repr
 
-    def make_samplers(self, device: torch.device, dtype: torch.dtype) -> None:
+    def make_samplers(self, device: Device = None, dtype: Dtype = None) -> None:
         if self.beta is None:
-            self._beta = torch.tensor(1.0, device=device, dtype=dtype)
+            self._beta = tensor(1.0, device=device, dtype=dtype)
```

Comment:
I would start considering to leverage the `Scalar` class im prototyping for single elements tensor (after is merged)

https://github.com/kornia/kornia/pull/1987/files#diff-42adf0d69e7dd1e765e13b264d372bd6aa41fd7e2e274edf2ce76ba05a6bb82fR12